app/experimenter/experiments/changelog_utils/nimbus.py

```diff
@@ -44,9 +44,14 @@ def generate_nimbus_changelog(experiment, changed_by, message):
 
     old_status = None
     old_publish_status = None
+    published_dto_changed = False
```

Comment:
Oh no it is, I should've explained better.  

- To check how long it's been since a change was pushed to kinto, we need the changelog at push time that updates publish status to waiting
- We had them for launch/end but not for pausing, so I added one (plus it's just nice to see in the changelog history when the pause change was pushed)
- Now a successful pause goes from LIVE/WAITING to LIVE/IDLE and a rejected pause goes from LIVE/WAITING to LIVE/IDLE, so we can't distinguish them
- The launch/end ones rely on the STATUS not changing and publish status going from WAITING > IDLE to know it's a rejection, but now we can't distinguish the accept/reject case for live updates
- So I thought about different ways to track it and checking whether the published dto changed seemed to be the simplest

Plus in the future we'll add other kinds of live updates like changing targeting, and this will help us with that later so, it solves this problem for now and I think will be useful later.